### PR TITLE
feat(ci): add pr labeler

### PR DESCRIPTION
**Pull request by c3ppc3pp**
This PR adds automatic labeling functions to PRs. It will scour the changeset and tries to make an educated guess about what label should be appropriate.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+'source:documentation': 'docs'
+'source:agent':
+  - 'golang/pkg/crane'
+  - 'golang/pkg/dagent'
+  - 'golang/cmd/crane'
+  - 'golang/cmd/dagent'
+'source:ci': '.github/workflows/'
+'source:build':
+  - '**/Makefile'
+  - 'web/kratos'
```
Comment (polaroi8d) [inline .github/labeler.yml:11]: We should add a `source:kratos` for that, IMHO. @m8vago your opinion?

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
```
Comment (polaroi8d) [inline .github/workflows/labeler.yml:1]: We are using the snake case convention to GitHub task names.

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+'source:web': 'web'
+'source:documentation': 'docs'
+'source:agent':
+  - 'golang/pkg/crane'
+  - 'golang/pkg/dagent'
+  - 'golang/cmd/crane'
+  - 'golang/cmd/dagent'
+'source:ci': '.github/workflows/'
+'source:build':
```
Comment (polaroi8d) [inline .github/labeler.yml:9]: Add `/images` folder for this, please.

**Review by polaroi8d — COMMENTED** [review]
Small improvements are needed, otherwise, if it works LGTM.

File: .github/workflows/labeler.yml (hunk 2 above)
Comment (c3ppc3pp) [reply .github/workflows/labeler.yml:1]: Not a single one had snake case :D but aight, changing all

Comment (c3ppc3pp) [reply .github/workflows/labeler.yml]: Done!

File: .github/labeler.yml (hunk 3 above)
Comment (c3ppc3pp) [reply .github/labeler.yml:9]: Fixed.

File: .github/labeler.yml (hunk 1 above)
Comment (c3ppc3pp) [reply .github/labeler.yml]: Done!

**Pull request merged**